samples/saga/ravendb-sequence-generator/Version_5/Sample/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NServiceBus;
+using NServiceBus.Persistence;
+using Raven.Client;
+using Raven.Client.Document;
+using System;
+using System.Threading.Tasks;
+
+class Program
+{
+
+    static void Main()
+    {
+        BusConfiguration busConfig = new BusConfiguration();
+		busConfig.EndpointName( "Samples.RavendbSequenceGenerator" );
+        busConfig.UseSerialization<JsonSerializer>();
+        busConfig.EnableInstallers();
```

Comment:
Oh yeah:-)

-----Original Message-----
From: "Simon Cropp" <notifications@github.com>
Sent: ‎23/‎03/‎2015 22:18
To: "Particular/docs.particular.net" <docs.particular.net@noreply.github.com>
Cc: "Mauro Servienti" <mauro.servienti@particular.net>
Subject: Re: [docs.particular.net] Using sagas to generate sequences (#589)

In samples/saga/ravendb-sequence-generator/Version_5/Sample/Program.cs:
> +﻿using NServiceBus;
> +using NServiceBus.Persistence;
> +using Raven.Client;
> +using Raven.Client.Document;
> +using System;
> +using System.Threading.Tasks;
> +
> +class Program
> +{
> +
> +    static void Main()
> +    {
> +        BusConfiguration busConfig = new BusConfiguration();
> +		busConfig.EndpointName( "Samples.RavendbSequenceGenerator" );
> +        busConfig.UseSerialization<JsonSerializer>();
> +        busConfig.EnableInstallers();
dont do that. I will add some common notes to every sample via the template that explains this
—
Reply to this email directly or view it on GitHub.